hermes-api/src/main/java/pl/allegro/tech/hermes/api/EndpointAddress.java

```diff
@@ -17,28 +17,14 @@
 @JsonSerialize(using = EndpointAddressSerializer.class)
 public class EndpointAddress {
 
-    private static final String ANONYMIZED_PASSWORD = "*****";
```

Comment:
Basic authorization is still there, but it is not written inside url. I have added select option "Authorization type" (3 options: NONE, BASIC, OAUTH2). When you choose BASIC, the username and password fields are displayed. So functionality remains.
Still you want to have it in the url?